app/views/api/v1/accounts/conversations/messages/index.json.jbuilder

```diff
@@ -3,6 +3,7 @@ json.meta do
   json.additional_attributes @conversation.additional_attributes
   json.contact @conversation.contact.push_event_data
   json.assignee @conversation.assignee.push_event_data if @conversation.assignee.present?
+  json.assignee @conversation.assignee.push_event_data if @conversation.assignee.present?
```

Comment:
duplicate. Did you mean json.team?

Reply:
@sojan-official Do we need team data here ?